agent/consul/leader.go

```diff
@@ -28,7 +28,18 @@ const (
 	barrierWriteTimeout = 2 * time.Minute
 )
 
-var minAutopilotVersion = version.Must(version.NewVersion("0.8.0"))
+var (
+	// caRootPruneInterval is how often we check for stale CARoots to remove.
+	caRootPruneInterval = time.Hour
+
+	// caRootExpireDuration is the duration after which an inactive root is considered
+	// "expired".
```

Comment:
Should we document why 7 days is a safe default for now? I guess once certificate lifetimes are configurable this will need to be dynamic based on that too?

Reply:
Yeah, it'll have to change - i'll update the comment for now though